src/libs/actions/IOU.js

```diff
@@ -65,6 +65,20 @@ Onyx.connect({
     },
 });
 
+let allPolicyTags = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_TAGS,
+    waitForCollectionCallback: true,
+    callback: (value) => {
+        if (!value) {
+            allPolicyTags = {};
```

Comment:
nit: seems like this is not needed?

Reply:
Ah, now I remember why, cause of the tests

Reply:
all good, you can leave it if needed!